### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by AruniAmarasinghe** [andpor/react-native-sqlite-storage#556]
Can we use this library for RN versions 0.69 and above?

**Comment by MacKenzieHnC** [comment]
No, it is not maintained. You gotta find the fork you need. If you plan on using Windows, you'll need my change, but it's like 2 lines of code you could easily add to any other fork.

I was successfully using it above 0.69 though.

**Comment by atiqulislamwb** [comment]
can you share this 2 line of code please and where i will paste @MacKenzieHnC 
i working on react native windows i face some errors

**Comment by MacKenzieHnC** [comment]
> can you share this 2 line of code please and where i will paste @MacKenzieHnC i working on react native windows i face some errors

[Here](https://github.com/andpor/react-native-sqlite-storage/pull/550)
